.github/workflows/build.yml

```diff
@@ -167,14 +167,14 @@ jobs:
         imageroot=$(find "${HOME}/JDK" -name release -type f)
         echo "TEST_JDK_HOME=$(dirname "${imageroot}")" >> "$GITHUB_ENV"
     - name: Smoke test
-      uses: adoptium/run-aqa@v2
+      uses: adoptium/run-aqa@6bacb4e732ad546eda1b09665b9067cdc87651f4 # v2
```

Comment:
maybe good to pin https://github.com/adoptium/run-aqa/releases/tag/v2.0.1 which is the latest (last week) released version?

Reply:
Yes, I did consider that, but wasn't sure that was intended as GitHub was still picking the existing v2.
@sophia-guo will it be OK for the smoke tests to use the run-aqa action at v2.0.1 ?

Reply:
v2.0.1 supports parallel, but it still needs improvement. For now I think v2 is good.